src/language-css/printer-postcss.js

```diff
@@ -48,6 +49,18 @@ const isSCSSMapNode = utils.isSCSSMapNode;
 const isDetachedRulesetCallNode = utils.isDetachedRulesetCallNode;
 const isPostcssSimpleVarNode = utils.isPostcssSimpleVarNode;
 
+function shouldPrintComma(options) {
+  switch (options.trailingComma) {
+    case "all":
+    case "es5":
+      return true;
+    // fallthrough
```

Comment:
You shouldn’t need this since `return` breaks out of the `switch`.

Reply:
@j-f1 done :+1: